committee-steering/governance/sig-governance.md

```diff
@@ -1,60 +1,134 @@
-  - Each SIG is assigned an update during the monthly community meeting throughout the year. The meeting host will publish the notes to the kubernetes-dev mailing list with the update.  
-  - If a SIG is doing an update during KubeCon + CloudNativeCon, the SIG Chairs should ensure that a link to the recording of the video is sent to the kubernetes-dev mailing list.  
-  - YouTube presentation or mailing list update sent to the kubernetes-dev mailing list.
-- Participate in release planning meetings and retrospectives, and burndown meetings, as needed
-- Ensure related work happens in a project-owned github org and repository, with code and tests explicitly owned and supported by the SIG, including issue triage, PR reviews, test-failure response, bug fixes, etc.
-- Use the [forums provided] as the primary means of working, communicating, and collaborating, as opposed to private emails and meetings
-
-The process for setting up a SIG or Working Group (WG) is listed in the [sig-wg-lifecycle] document.
+- Meet regularly, at least for 30 minutes every 3 weeks, except November and
+December
+- Keep up-to-date meeting notes, linked from the SIG's page in the community
+repo
+- Ensure other SIGs and community members can find notes of meetings,
+discussions, designs, and decisions
+- Record meetings and make them publicly available on the
+[Kubernetes Community YouTube playlist]
+- Report activity with the community via the kubernetes-dev mailing list at
+least once a quarter. Whichever format the SIG uses should _always_ be reported
+to the kubernetes-dev mailing list as this is the list the community depends on
+for SIG updates.  
+  - Each SIG is assigned an update during the monthly community meeting
+  throughout the year. The meeting host will publish the notes to the
+  kubernetes-dev mailing list with the update.  
+  - If a SIG is doing an update during KubeCon + CloudNativeCon, the SIG Chairs
+  should ensure that a link to the recording of the video is sent to the
+  kubernetes-dev mailing list.  
+  - YouTube presentation or mailing list update sent to the kubernetes-dev
+  mailing list.
+- Participate in release planning meetings and retrospectives, and burndown
+meetings, as needed
+- Ensure related work happens in a project-owned github org and repository, with
+ code and tests explicitly owned and supported by the SIG, including issue
+ triage, PR reviews, test-failure response, bug fixes, etc.
+- Use the [forums provided] as the primary means of working, communicating, and
+collaborating, as opposed to private emails and meetings  
+- SIG overview and deep-dive sessions organized for KubeCon/CloudNativeCon
+- Contributing instructions (CONTRIBUTING.md) defined in the SIGs folder located
+ in the Kubernetes/community repo  
```

Comment:
Can we rephrase this to start with something like "Ensure contributing instructions  (CONTRIBUTING.md) defined in..."?

Most SIGs don't have this file right now. Is this something that SIGs _MUST_ do or is it something SIGs _MAY_?

Reply:
PTAL